stylesheets/login.css

```diff
@@ -0,0 +1,49 @@
+.login-container {
+	width: 370px;
+	margin: auto;
+	margin-top: 6rem;
+	height: fit-content;
+	padding: 15px;
+	border-radius: 5px;
+	box-shadow: var(--dark-box-shadow);
+	display: flex;
+	flex-direction: column;
+}
+
+.login-container input {
+	width: auto;
+	border-radius: 4px;
+	font-size: 0.8rem;
+	font-weight: 500;
```

Comment:
You can declare variables for font weight and margin etc too

Reply:
ok , noted